packages/ToggleSwitch/ToggleSwitch.jsx

```diff
@@ -57,23 +65,33 @@ class ToggleSwitch extends Component {
   }
 
   render() {
-    const { id, label, name, value, checked, onBlur, onChange, onFocus, ...rest } = this.props
-
+    const {
+      id,
+      label,
+      name,
+      value,
+      toolTipText,
+      checked,
+      onBlur,
+      onChange,
+      onFocus,
+      toolTipCopy,
+      isLoading,
+      ...rest
+    } = this.props
     const labelledById = `${id}-label`
     const disabled = !!rest && !!rest.disabled
 
-    const switchStatusClass = this.state.checked ? styles.switchOn : styles.switchOff
-    const switchClasses = disabled
-      ? [styles.switch, switchStatusClass, styles.switchDisabled]
-      : [styles.switch, switchStatusClass]
-
     return (
```

Comment:
You can disable this eslint rule for now by adding this above this `return` statement:

`/* eslint-disable jsx-a11y/label-has-for */`

This eslint rule got deprecated and needs to be replaced with a different rule. Since our team will be moving to `telus-standard` - a different linting tool it should be safe to disable this rule for now.

Reply:
added